addons/stock/wizard/product_replenish.py

```diff
@@ -76,6 +82,22 @@ def launch_replenishment(self):
                     self._prepare_run_values()  # Values
                 )
             ])
+            if len(procurement_created.values()) == 1:
+                for record in procurement_created.values():
```

Comment:
you can remove the loop if there as there is only 1 value

Reply:
`procurement_created.values()` result is coming like this `dict_values([purchase.order(12,)])` we can't grab this by indexing so we added a for loop.

Instead of that, we can get It like this without for loop
`record = list(procurement_created.values())[0]`

Reply:
We should reuse the same notification mechanism we have in the orderpoint instead of passing the result in all the `run_*` methods. See 
https://github.com/odoo/odoo/blob/5a5a00a3528e51d986eb62d619c2eea13df575ca/addons/stock/models/stock_orderpoint.py#L231-L232

Also, could you clean your commits and write a detailed commit message. Thank you

Reply:
Hello @Whenrow  we can't, cause we don't have `orderpoint` here, to track which route is performed.